Week1/homework/app.js

```diff
@@ -3,9 +3,162 @@
 {
   const bookTitles = [
     // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+    'moby_dick',
+    'the_adventures_of_huckleberry_finn',
+    'alices_adventures_in_wonderland',
+    'charlie_and_the_chocolate_factory',
+    'wuthering_heights',
+    'war_and_peace',
+    'anna_karenina',
+    'animal_farm',
+    'a_tale_of_two_cities',
+    'the_alchemist'
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
+  const Books = {
+    moby_dick: {
+      title: 'Moby-Dick',
+      author: 'Herman Melville',
```

Comment:
I would like to appreciate that you followed the instructions while naming properties for your `books` object and not deviating from it.

Would like to add one small thing here which you might learn a little later. ES6 has introduced a new feature called [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). This makes use of back-ticks and a couple of advantages of using them is that you can also include variables inside your strings and split really long lines as we have in `info` property in multi-line strings which are then easier to read. 🙂